templates/org/create.tmpl

```diff
@@ -37,7 +37,7 @@
 						<label>{{.locale.Tr "org.settings.permission"}}</label>
 						<div class="inline-grouped-list">
 							<div class="ui checkbox">
-								<input class="hidden" type="checkbox" name="repo_admin_change_team_access" checked/>
+								<input class="hidden" type="checkbox" name="repo_admin_change_team_access" {{if or (not .Err_OrgName) .repo_admin_change_team_access}}checked{{end}}/>
```

Comment:
There could be other errors, not only `Err_OrgName`.

Maybe
```suggestion
								<input class="hidden" type="checkbox" name="repo_admin_change_team_access" {{if not (eq .repo_admin_change_team_access false)}}checked{{end}}/>
```

So checked only if `.repo_admin_change_team_access` is nil (the new page) or is true (the posted page with it checked)